### PR TITLE
added information for gcp-weather node

### PR DESCRIPTION
**Pull request by PatrickAlphaC**

### CODE REVIEW

File: docs/Data Provider Nodes/google-weather.md [hunk 1]
```diff
@@ -0,0 +1,361 @@
+---
+layout: nodes.liquid
+section: smartContract
+date: Last Modified
+title: "Google"
```
Comment (StephenFluin) [inline docs/Data Provider Nodes/google-weather.md:5]: Could we make the title more specific about what this page is?

**Review by StephenFluin — CHANGES_REQUESTED** [review]
Should this be listed on the navigation or the list at https://deploy-preview-287--dreamy-villani-0e9e5c.netlify.app/docs/data-provider-nodes/ ?

**Comment by bryanjowers** [comment]
Looking at preview, looks good. Do we still have a plan for indicating that this kind of data is only sourced from a single data provider / a single Oracle that is providing responses?

File: docs/Data Provider Nodes/google-weather.md (hunk 1 above)
Comment (PatrickAlphaC) [reply docs/Data Provider Nodes/google-weather.md:5]: Wow, totally spaced on that. Thank you!

**Comment by PatrickAlphaC** [comment]
@bryanjowers this doc is in the data node provider section of the docs. We used to have a section that clarified this, although it can be easily implied by the descriptions as well. I'll defer to @StephenFluin on why we took them out in the first place.

File: docs/Data Provider Nodes/google-weather.md (hunk 1 above)
Comment (PatrickAlphaC) [reply docs/Data Provider Nodes/google-weather.md]: And yes, it should be in that section!

**Comment by PatrickAlphaC** [comment]
@bryanjowers the section here: https://docs.chain.link/docs/data-provider-nodes/ explains the function of these types of oracles

**Review by StephenFluin — APPROVED** [review]

**Pull request merged**